### ISSUE TITLE
Free license

### ISSUE THREAD

**Issue by metah4cker** [rootmanX/whatsapp-xroot#1]
We had hacked your tool! Everyone can use this license code: `Lj23FQxxwr`.

**Comment by avr007** [comment]
> Lj23FQxxwr

Thank you bro, this is an Indian scambaiter!, this tool doesn't display anything other than WhatsApp web without logging in !

**Comment by metah4cker** [comment]
> > Lj23FQxxwr
> 
> Thank you bro, this is an Indian scambaiter!, this tool doesn't display anything other than WhatsApp web without logging in !

Yes, you're right. This is really fake tool developed for scamming. Author didn’t even bother to hide it, let alone the skill of creating such a powerful tool. Funny :)

**Comment by stewsoupbrown** [comment]
> > > Lj23FQxxwr
> > 
> > 
> > Thank you bro, this is an Indian scambaiter!, this tool doesn't display anything other than WhatsApp web without logging in !
> 
> Yes, you're right. This is really fake tool developed for scamming. Author didn’t even bother to source code, let alone the skill of creating such a powerful tool. Funny :)

so it doesnt work at all?
